client/src/modules/reports/generate/inventoryChange/inventoryChange.config.js

```diff
@@ -0,0 +1,76 @@
+angular.module('bhima.controllers')
+  .controller('inventoryChangeController', InventoryChangeController);
+
+InventoryChangeController.$inject = [
+  '$sce', 'NotifyService', 'BaseReportService',
+  'AppCache', 'reportData', '$state',
+];
+
+function InventoryChangeController($sce, Notify, SavedReports, AppCache, reportData, $state) {
+  const vm = this;
+  const cache = new AppCache('inventoryChange');
+  const reportUrl = 'reports/inventory/changes/';
+
+  vm.previewGenerated = false;
+  vm.reportDetails = {
+    shouldShowRemainDetails : 1,
+    shouldShowInvoicedDetails : 1,
+    shouldShowPaidDetails : 1,
+  };
+
+  vm.clearPreview = function clearPreview() {
+    vm.previewGenerated = false;
+    vm.previewResult = null;
+  };
+
+  vm.onChangeShowRemainDetails = value => {
+    vm.reportDetails.shouldShowRemainDetails = value;
+  };
+
+  vm.onChangeShowPaidDetails = value => {
+    vm.reportDetails.shouldShowPaidDetails = value;
+  };
+
+  vm.onChangeShowInvoicedDetails = value => {
```

Comment:
It doesn't look like you use any of this.  Please remove anything you don't use or need.